### DISCUSSION TITLE
How can I get the URL fragment?

### DISCUSSION THREAD

**Discussion by bnctth** [sveltejs/kit#3002] [Q&A] [answered]
I can't seem to find a way to extract the URL fragment (the thing after the #) anywhere. I'm trying to get it in a load function (or hook if necessary). Thanks

**Comment by machak** [comment, marked as answer]
if you want to get it on the server (SSR), that's a no, client side you can get it via url.hash I believe. 

"The search part, if present, is
   sent as part of the HTTP command, and may in this respect be
   treated as part of the path. **No fragmentid part of a WWW URI (the
   hash sign and following) is  sent with the request.**   Spaces and
   control characters in URLs must be escaped for transmission in
   HTTP, as must other disallowed characters. "
